src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -568,6 +568,13 @@ def get_active_daemon(self, daemon_descrs: List[DaemonDescription]) -> DaemonDes
         # if no mds found, return empty Daemon Desc
         return DaemonDescription()
 
+    def purge(self, service_name: str) -> None:
+        self.mgr.check_mon_command({
+            'prefix': 'config rm',
+            'who': service_name,
+            'name': 'mds_join_fs',
+        })
```

Comment:
hm. Feels as if we should ignore errors here. thoughts?

Reply:
What kind of errors are you expecting here other than if the provided service name is wrong?

Reply:
things like someone removed `mds_join_fs` manually. I have no concrete idea. But I#d really like to prevent an infinite loop of always trying to purge this service and failing every time. Sound like a great way to make people really scared.